susemanager-utils/susemanager-sls/src/modules/appstreams.py

```diff
@@ -92,6 +92,26 @@ def _get_module_info(module_names):
         return []
 
     module_info_output = result.stdout.splitlines()
+    active_modules = []
```

Comment:
We should also change this: https://github.com/uyuni-project/uyuni/pull/9372/files#diff-0afe89c35f776dab892a3a79e2f20c5db733ec5c7c2002251a6b205145e7bf43L86 as now the function get the info for active modules only.

Also, before starting parsing the output, I would add an example output to understand better what the loop is trying parse/achieve.